config/300-customrun.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: customruns.tekton.dev
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+    pipeline.tekton.dev/release: "devel"
+    version: "devel"
+spec:
+  group: tekton.dev
+  preserveUnknownFields: false
+  versions:
+  - name: v1beta1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        type: object
+        # One can use x-kubernetes-preserve-unknown-fields: true
+        # at the root of the schema (and inside any properties, additionalProperties)
+        # to get the traditional CRD behaviour that nothing is pruned, despite
+        # setting spec.preserveUnknownProperties: false.
+        #
+        # See https://kubernetes.io/blog/2019/06/20/crd-structural-schema/
+        # See issue: https://github.com/knative/serving/issues/912
+        x-kubernetes-preserve-unknown-fields: true
+    additionalPrinterColumns:
+    - name: Succeeded
+      type: string
+      jsonPath: ".status.conditions[?(@.type==\"Succeeded\")].status"
+    - name: Reason
+      type: string
+      jsonPath: ".status.conditions[?(@.type==\"Succeeded\")].reason"
+    - name: StartTime
+      type: date
+      jsonPath: .status.startTime
+    - name: CompletionTime
+      type: date
+      jsonPath: .status.completionTime
+    # Opt into the status subresource so metadata.generation
+    # starts to increment
+    subresources:
+      status: {}
+  names:
```

Comment:
optional: add a shortname?

Reply:
Does `cr` sound alright?

Reply:
yeah sg!

Reply:
(worth mentioning that `cr` is a short name for `cert-manager.io/v1`'s `CertificateRequest` - which is common enough that collisions/overlaps in the same cluster seem likely)

Reply:
Yeah names collisions are fun. If you have knative and tekton installed on the same cluster, the only way to get a tekton trigger is a `trigger.triggers.tekton.dev`... as `trigger` is understood as the knative one...

What about using `run` as short name? Since we are going to remove the alpha `Run` it won't conflict I think

Reply:
I'm going to let this discussion go on for a bit to see if we can settle on a consensus, since I'm not aiming to merge this til post v0.41.0 anyway. =)

That said, if we want to go with `run` as the short name, we'd need to wait on doing so until `v1alpha1.Run` is removed, I think, to avoid confusion.

Reply:
We could start without shortnames and take time to figure out one if we need to.